data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/AsyncCloseable.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
```

Comment:
Do we actually need to provide a date? Or worth to update the templates on the IDE ? :smile:

Reply:
From what I know, this template is fine and legal friendly, because it is the starting date of the project :smile: